### DISCUSSION TITLE
GitHub Enterprise Privacy Updates - Universe 2024

### DISCUSSION THREAD

**Discussion by queenofcorgis** [community/community#142193] [Enterprise]
![unnamed](https://github.com/user-attachments/assets/66bbbeac-b07b-4912-ad07-fc6ad33c2a49)

It’s no surprise that we bring engineering and security together in everything we build, in case you missed the announcement, we revealed new privacy releases for GitHub Enterprise. 

Read on for the highlights and how you can keep your GitHub more secure. 

### Now generally available: EU Data Residency for GitHub Enterprise 

GitHub Enterprise Cloud empowers you with the flexibility to choose where your code is stored, starting with the EU and expanding to more regions in the future. This enhanced control allows you to manage your data residency preferences to meet the unique needs of your business—whether for compliance, performance, availability, or other reasons. Powered by Microsoft Azure's enterprise-grade infrastructure and security, your code is protected both in transit and at rest, benefiting from Microsoft's high standards and operational excellence.

GitHub Enterprise Cloud is available to customers seeking a comprehensive enterprise-grade development platform with the scalability of a cloud-based solution and the flexibility to control where their code and repository data reside. We are starting with general availability in the EU with additional regions planned for the near future.

[Read More here](https://github.blog/changelog/2024-10-29-github-enterprise-cloud-data-residency-in-the-eu-is-generally-available).

Security is an ongoing conversation, if you’d like to join the conversations on security with our speakers from GitHub Universe 2024, we recommend the following discussions: 
- [Enterprise code security 101: Getting started with GitHub Advanced Security](https://github.com/orgs/community/discussions/139507)
- [Why (and how) IKEA, WirelessCar, and SAP adopted GitHub Advanced Security](https://github.com/orgs/community/discussions/141572) 
- [Go big! Efficiently deploy and customize security tooling at enterprise scale](https://github.com/orgs/community/discussions/141479)

And if you’re looking to up-level your own security knowledge, take the [GitHub Advanced Security certification](https://resources.github.com/learn/certifications/), we even have a [prep course on the community](https://github.com/orgs/community/discussions/137197) to help you study.

**Comment by khitrenovich** [comment]
Is this available for EMU only?

**Reply by queenofcorgis** [reply]
@khitrenovich thanks for your patience, yes it is available for EMU only as it's based on the EMU version of GitHub Enterprise Cloud on dotcom.

**Comment by Nephew209** [comment]
I have a opinion about this issue if a individual is involved in multiple organizations maybe implying enterprise already has it's value set I would suggest ether maybe changing it or maybe granting  or permission to that individual involved with the organizations and be able modify it so they are able to collaborate thus being able to cut costs and possibly create a single organization under the same principles
